data/types/base.go

```diff
@@ -35,7 +35,7 @@ const (
 	TimeZoneOffsetMaximum   = 7 * 24 * 60  // TODO: Fix! Limit to reasonable values
 	TimeZoneOffsetMinimum   = -7 * 24 * 60 // TODO: Fix! Limit to reasonable values
```

Comment:
Maybe not in this PR:
```suggestion
	TimeZoneOffsetMaximum   = 14 * 24 * 60
	TimeZoneOffsetMinimum   = -12 * 24 * 60
```
https://en.wikipedia.org/wiki/List_of_UTC_time_offsets

Reply:
agree that it's not in the scope of this PR ;-)